integrations/astra/src/haystack_integrations/components/retrievers/astra/retriever.py

```diff
@@ -31,14 +33,25 @@ class AstraEmbeddingRetriever:
     ```
     """
 
-    def __init__(self, document_store: AstraDocumentStore, filters: Optional[Dict[str, Any]] = None, top_k: int = 10):
+    def __init__(
+        self,
+        document_store: AstraDocumentStore,
+        filters: Optional[Dict[str, Any]] = None,
+        top_k: int = 10,
+        filter_policy: Optional[Union[str, FilterPolicy]] = FilterPolicy.REPLACE,
```

Comment:
```suggestion
        filter_policy: Union[str, FilterPolicy] = FilterPolicy.REPLACE,
```

My impression is that param is not `Optional`: the user can provide a (not null) value; otherwise we have a default.